src/libraries/System.Net.Security/src/System/Net/Security/Pal.OSX/SafeDeleteSslContext.cs

```diff
@@ -158,13 +169,24 @@ private unsafe int WriteToConnection(void* connection, byte* data, void** dataLe
             ulong length = (ulong)*dataLength;
             Debug.Assert(length <= int.MaxValue);
 
+            if (Interlocked.Exchange(ref _pendingOutput, 1) == 2)
+            {
+                const int writErr = -20;
```

Comment:
Where does this -20 and the below -19 numbers come from?  Are those arbitrary error values, or do they map to something else?

Reply:
they are osStatus enums
https://www.osstatus.com/search/results?platform=all&framework=all&search=-19

same as noErr below.

Reply:
Could you rename then to `const int OSStatus_writErr` or something like that?

Or maybe introduce an OSStatus enum somewhere under interop?

Reply:
It seems like this does not show up in other places as we map the actual values in PAL via
https://github.com/dotnet/runtime/blob/d6ea28dd585ae9366fd9878264157c4acaf410a1/src/libraries/Native/Unix/System.Security.Cryptography.Native.Apple/pal_ssl.c#L278-L280

Looking at this again, I can probably use errSSLClosedAbort = -9806 for both and move it up to top (with noErr) and add comment to it. And use the `OSStatus_` prefix.  Since this is disposed, I don't think the actually value matters that much.